worklets/Overview.bs

```diff
@@ -259,52 +269,84 @@ Note: As an example the <a>worklet global scope type</a> might be a {{PaintWorkl
 A {{Worklet}} has a list of the <dfn export>worklet's WorkletGlobalScopes</dfn>. Initially this list
 is empty; it is populated when the user agent chooses to create its {{WorkletGlobalScope}}.
 
-A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
-empty; it is populated when module scripts resolved.
+A {{Worklet}} has a <dfn>module responses map</dfn>. This is a ordered map of module URLs to values
+that are a <a>fetch</a> responses. The map's entries are ordered based on their insertion order.
+Access to this map should be thread-safe.
+
+The <a>module responses map</a> exists to ensure that {{WorkletGlobalScope}}s created at different
+times contain the same set of script source text and have the same behaviour. The creation of
+additional {{WorkletGlobalScope}}s should be transparent to the author.
 
 When the <dfn method for=Worklet>import(|moduleURL|)</dfn> method is called on a {{Worklet}} object,
 the user agent <em>must</em> run the following steps:
     1. Let |promise| be <a>a new promise</a>.
 
-    2. Run the following steps <a>in parallel</a>:
-        1. Let |resolvedModuleURL| be the result of <a>resolving</a> the |moduleURL| relative to the
-            <a>API base URL</a> specified by the <a>entry settings object</a> when the method was
-            invoked.
+    2. Let |resolvedModuleURL| be the result of <a>parsing</a> the |moduleURL| argument relative to
+        the <a>settings object</a> of <b>this</b>.
```

Comment:
You want [relevant settings object](https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object) of `this`.